ee/settings.py

```diff
@@ -15,8 +15,6 @@
     "action_performed": "posthog.Action.performed",
     "annotation_created": "posthog.Annotation.created_custom",
 }
-HOOK_FINDER = "ee.models.hook.find_and_fire_hook"
-HOOK_DELIVERER = "ee.models.hook.deliver_hook_wrapper"
```

Comment:
This is used by `django-rest-hooks`, the library we use to integrate with Zapier. Specifically, this fires our Zapier integration's Triggers. However, the only trigger that actually matters is the "action_performed" one, and that has been completely delegated to the plugin server for a while. We _can_ remove this, along with `django-rest-hooks`, but first we need to make sure `action_defined` and `annotation_created` actually are unused, and roll out an update to https://github.com/PostHog/posthog-zapier that'd remove those two completely.

Reply:
TIL. Thank you. I'm closing in favour of #9146. I'll create an issue to do a proper cleanup.